index.html

```diff
@@ -53,18 +60,21 @@ <h3>LETS CONNECT</h3>
             </ul>
           </div>
         </div>
-      </div>
-    </section>
+       <!-- </div> -->
+     </section>
+    </div>
+    </main>
     <section class="works">
```

Comment:
- [ ] [OPTIONAL] The project cards are stretched and have less padding, please consider making changes to the design so it can be as close as possible to the Figma one

|Your design|Figma design|
|-|-|
|![Screenshot_20220603_214110](https://user-images.githubusercontent.com/25227022/171949622-856575e2-2bb3-44f9-ae69-70c6fd0071d4.png)|![Screenshot_20220603_214148](https://user-images.githubusercontent.com/25227022/171949642-ba3fad2d-ffe3-4dbf-90ca-3bdc9e218f62.png)|